backend/src/main/java/com/dhbw/tutorsystem/role/RoleRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.dhbw.tutorsystem.role;
+
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.Optional;
+
+public interface RoleRepository extends CrudRepository<Role, Integer> {
+    Optional<Role> findByName(ERole name);
```

Comment:
Is the name the real name or the email?

Reply:
Please clarify

Reply:
Fixed [400b1d0](https://github.com/FinnHuelsbusch/tutorensystem/pull/9/commits/400b1d013a14af47fc7e4d0b76474ee5c82c4b39)